fiftyone/core/cli.py

```diff
@@ -2706,9 +2705,8 @@ def _print_operators_list(enabled, names_only):
                 "uri": op.uri,
                 "enabled": op.builtin or op.plugin_name in enabled_plugins,
                 "builtin": op.builtin,
-                "on_startup": op.config.on_startup,
                 "unlisted": op.config.unlisted,
-                "dynamic": op.config.dynamic,
+                "on_startup": op.config.on_startup,
```

Comment:
This change doesn't make sense... I'm sure there's some reason. Perhaps its that dynamic isn't really an externally interesting attribute..?

Reply:
I removed it because the output tables I was putting in the docs wouldn't fit without line breaking, lol.

But also yes `dynamic` is an internal thing that's not useful to see in the CLI output. win-win